pkgs/development/python-modules/mapbox-earcut/tests.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/tests/test_earcut.py b/tests/test_earcut.py
```

Comment:
Same comment as above, especially since this is a repeating pattern.
```bash
substituteInPlace tests/test_earcut.py --replace \
   "pytest.raises(ValueError, message(\"Expecting ValueError\")" "with pytest.raises(ValueError)"

Reply:
done!